.cloudbuild.yaml

```diff
@@ -198,6 +198,47 @@ steps:
   args: ['pull', 'gcr.io/ml-pipeline/workflow-controller:v3.2.3-license-compliance']
   id:   'pullArgoWorkflowController'
 
+# V2 related images
+# Prerequisite: Make sure ko image is available on the same project by running the following:
+# git clone https://github.com/GoogleCloudPlatform/cloud-builders-community.git
+# cd cloud-builders-community/ko
+# gcloud builds submit . --config=cloudbuild.yaml --project=$PROJECT_ID
+# Reference: https://dev.to/amammay/effective-go-on-gcp-lean-containers-with-ko-on-cloud-build-51ek
+
+- name: 'gcr.io/$PROJECT_ID/ko'
+  entrypoint: /bin/sh
+  args:
+    - -c
+    - |
+      cd /workspace/v2/
+      /ko publish --bare ./cmd/launcher-v2 -t $COMMIT_SHA
+  env:
+  - 'KO_DOCKER_REPO=gcr.io/$PROJECT_ID/dev/kfp-launcher-v2'
+  id:   'buildLauncherV2'
+  waitFor: ["-"]
+- name: 'gcr.io/$PROJECT_ID/ko'
+  entrypoint: /bin/sh
+  args:
+    - -c
+    - |
+      cd /workspace/v2/
+      /ko publish --bare ./cmd/driver -t $COMMIT_SHA
+  env:
+  - 'KO_DOCKER_REPO=gcr.io/$PROJECT_ID/dev/kfp-driver'
+  id:   'buildDriver'
+  waitFor: ["-"]
+- name: 'gcr.io/$PROJECT_ID/ko'
+  entrypoint: /bin/sh
+  args:
+    - -c
+    - |
+      cd /workspace/v2/
+      /ko publish --bare ./cmd/compiler -t $COMMIT_SHA
+  env:
+  - 'KO_DOCKER_REPO=gcr.io/$PROJECT_ID/dev/kfp-compiler'
+  id:   'buildCompiler'
+  waitFor: ["-"]
```

Comment:
For now, we don't need to use the compiler as an image.

Two current use-cases:
* KFP backend imports compiler as a library
* KFP v2 sample test builds the compiler as a binary and put it in sample test image

Reply:
Thank you for the background! Removed.